O/open62541/build_tarballs.jl

```diff
@@ -3,17 +3,24 @@
 using BinaryBuilder, Pkg
 
 name = "open62541"
-version = v"1.3.9"
+version = v"1.4.0"
 
 # Collection of sources required to complete build
 sources = [
     GitSource("https://github.com/open62541/open62541.git",
-              "70ff3501ddecd7e7594ebc63e2365994d59e010d")
+              "84347820c8550b5750f2cd581c14ab201611c579"),
+    DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+# Necessary for cmake to find openssl on Windows 
+if [[ ${target} == x86_64-*-mingw* ]]; then 
+    export OPENSSL_ROOT_DIR=${prefix}/lib64 
+fi 
+
 cd $WORKSPACE/srcdir/open62541/
+atomic_patch -p1 ../0001-freebsd.patch
```

Comment:
In general I prefer applying patches to all platforms when possible, to avoid platform-specific differences, but this patch makes me slightly uncomfortable, so let's apply it only for FreeBSD until there's a proper fix upstream  
```suggestion
if [[ "${target}" == *-freebsd* ]]; then
    # https://github.com/open62541/open62541/issues/6414
    atomic_patch -p1 ../0001-freebsd.patch
fi
```